README.md

```diff
@@ -48,365 +45,17 @@ First, you will need to customize and improve your document retrieval project on
 
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - [nvm](https://github.com/nvm-sh/nvm#installation-and-update)
-  - Set Node version to 16.x.x via `nvm install 16` and `nvm use 16`
+  - Set Node version to 16.x.x via `nvm install 16` and `nvm use 16` (also compatible with Node 18)
```

Comment:
Did you verify that it works as expected with Node 18?

Reply:
Yes